renovate.json

```diff
@@ -21,7 +33,6 @@
         "nodemon"
       ],
       "automerge": true,
-      "automergeStrategy": "squash",
```

Comment:
?

Reply:
I've set it at the top-level which hopefully works and means not repeating it every time